### ISSUE TITLE
can we please make a release of deadpool redis?

### ISSUE THREAD

**Issue by suryapandian** [bikeshedder/deadpool#309]
when we updated `redis` 0.25
without the updated deadpool, we get 

```
Caused by:
    0: cannot build redis client
    1: Unexpected redis internal error
    2: can't connect with TLS, the feature is not enabled- InvalidClientConfig
```

**Comment by bikeshedder** [comment]
I'll release the new `deadpool` redis crates shortly.

You should be able to enable features by adding the `redis` crate in version `0.24` and enabling the features there.

This is also explained in the FAQ: https://crates.io/crates/deadpool-redis#faq

**Comment by suryapandian** [comment]
yes, while we use `deadpool-redis` with `0.24` of `redis` works well.
We are not able to update `redis` since `deadpool` breaks.

**Comment by omer-shtivi** [comment]
Hello,
We are also waiting for the release, we need version 0.25 of Redis and we are unable to upgrade.

Thanks

**Comment by bikeshedder** [comment]
I just released `deadpool-redis 0.15.0` on crates.io:

- https://crates.io/crates/deadpool-redis/0.15.0

**Comment by suryapandian** [comment]
> I just released `deadpool-redis 0.15.0` on crates.io:
> 
>     * https://crates.io/crates/deadpool-redis/0.15.0

Thanks a lot!
